src/routes/datasets/[id].svelte

```diff
@@ -28,7 +29,8 @@
 	function doQuery(_query) {
 		const endpoint = datasetInfo.spec.dataset.endpoint_url;
 		const url = `${endpoint}/_search`;
-		return request(fetch, 'POST', url, {data: _query});
+
+		return IS_BROWSER && request(fetch, 'POST', url, {data: _query});
```

Comment:
What happens downstream at promise resolve time when it's not a browser and it's expecting a response object?

Reply:
It returns `false` which gets managed correctly by `svelte-json-tree`